### ISSUE TITLE
Filling out the curses stub

### ISSUE THREAD

**Issue by gotyaoi** [python/typeshed#789]
I'm doing a little work with curses recently, and noticed that the curses stub was all dynamic, so I gave a shot at filling some of it in. It needs a bit more work, but would this be useful to continue, and am I doing it correctly? 

https://github.com/gotyaoi/typeshed/commit/e3dbef0714989d24e93b3161b8912555e7fe3325

**Comment by ambv** [comment]
Absolutely! Thank you for working on this. What you currently have already looks like a good improvement. You can cross-check your types with the library's documentation and Python's own unit tests for the library.

**Comment by gotyaoi** [comment]
The tests didn't seem to assert much, so I just went by what was in _cursesmodule.c. https://github.com/gotyaoi/typeshed/commit/f5a88096fb421c26a7e374b4945eec3c08cc35d3

I cleaned up a few things, but I'm a little stuck on what to do about the window objects. They seem to fall under this issue: https://github.com/python/typeshed/issues/24. I'm reasonably sure I could document them, but I don't know what to call the class...
